autorelease/trigger.py

```diff
@@ -89,7 +91,51 @@ def trigger_kokoro_build_for_pull_request(
         sha=sha,
         env_vars={"AUTORELEASE_PR": pull_request_url},
     )
-    gh.update_pull_labels(pull, add=["autorelease: triggered"])
+    if update_labels:
+        gh.update_pull_labels(pull, add=["autorelease: triggered"])
+
+
+def _parse_issue(pull_request_url: str) -> Tuple[str, int]:
+    match = re.match(".*github.com/(.*)/pull/(\\d+)", pull_request_url)
+    if match:
+        return match[1], match[2]
+    raise Exception(f"bad url format: {pull_request_url}")
+
+
+def trigger_single(
+    github_token: str, kokoro_credentials: str, pull_request_url: str
+) -> reporter.Reporter:
+    report = reporter.Reporter("autorelease.trigger")
+    # TODO(busunkim): Use proxy once KMS setup is complete.
```

Comment:
This TODO looks stale?

Reply:
It's in all the autorelease jobs and they should all be addressed together